client/my-sites/domains/domain-management/transfer/transfer-to-other-user/transfer-domain-to-other-user.jsx

```diff
@@ -313,13 +313,13 @@ class TransferDomainToOtherUser extends Component {
 				<>
 					<p>
 						{ translate(
-							'Please choose an administrator to transfer domain mapping of {{strong}}%(domainName)s{{/strong}} to.',
+							'Please choose an administrator to transfer domain connection of {{strong}}%(domainName)s{{/strong}} to.',
 							{ args: { domainName }, components: { strong: <strong /> } }
 						) }
 					</p>
 					<p>
 						{ translate(
-							'You can transfer this domain mapping to any administrator on this site. If the user you want to ' +
+							'You can transfer this domain connection to any administrator on this site. If the user you want to ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'You can transfer this domain to any administrator on this site. If the user you want to transfer is not currently an administrator, please {{a}}add them to the site first{{/a}}.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domain-transfer-dialog-text)